backend/init_db.py

```
@@ -75,6 +75,14 @@ def migrate_db():
                 # Index likely already exists
                 pass
 
+            # Add composite index for spatial queries with status filter
+            try:
+                conn.execute(text("CREATE INDEX ix_issues_status_lat_lon ON issues (status, latitude, longitude)"))
+                logger.info("Migrated database: Added composite index on status, latitude, longitude.")
+            except Exception:
+                # Index likely already exists
+                pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)